### PR TITLE
DM-25777: Add ts_simactautors to dependencies

### PR DESCRIPTION
**Pull request by r-owen**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ test:
     - ts-dds
     - ts-idl
     - ts-salobj
+    - ts-simactuators
     - black
   source_files:
     - python
```
Comment (tribeiro) [inline conda/meta.yaml:39]: Please also add `ts-simactuators` and `ts-idl` as a "run" dependency.

**Review by tribeiro — COMMENTED** [review]
Please, add `LSST_DDS_DOMAIN` to the list of env variables exposed to scons unit test:

https://github.com/lsst-ts/ts_ATDomeTrajectory/blob/f38212d0ca4e1816099fc6b2216311ac2d2d8a80/tests/SConscript#L8

**Comment by r-owen** [comment]
Why do you want LSST_DDS_DOMAIN passed to the unit tests? Every test must set a random value for LSST_DDS_DOMAIN. If it does not then that is a serious bug that could cause collisions with other users. I think it is dangerous to pass that value to the tests.

**Comment by tribeiro** [comment]
I don't know, but the tests won't pass when you run `scons` if `LSST_DDS_DOMAIN` is not passed....

**Comment by tribeiro** [comment]
This is what I get if I run scons without passing `LSST_DDS_DOMAIN`:

```
==================================================================================== FAILURES ====================================================================================
____________________________________________________________________ ATDomeTrajectoryTestCase.test_bin_script ____________________________________________________________________

self = <test_csc.ATDomeTrajectoryTestCase testMethod=test_bin_script>

    async def test_bin_script(self):
        """Test that run_atdometrajectory.py runs the CSC.
        """
        await self.check_bin_script(
>           name="ATDomeTrajectory", index=None, exe_name="run_atdometrajectory.py",
        )

tests/test_csc.py:68:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/opt/lsst/software/stack/stack/tssw/ts_salobj/python/lsst/ts/salobj/base_csc_test_case.py:247: in check_bin_script
    domain=domain, name=name, index=index
/opt/lsst/software/stack/stack/tssw/ts_salobj/python/lsst/ts/salobj/remote.py:154: in __init__
    salinfo = SalInfo(domain=domain, name=name, index=index)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = SalBase(ATDomeTrajectory, 0), domain = <lsst.ts.salobj.domain.Domain object at 0x7f6a30380210>, name = 'ATDomeTrajectory', index = None

    def __init__(self, domain, name, index=0):
        if not isinstance(domain, Domain):
            raise TypeError(f"domain {domain!r} must be an lsst.ts.salobj.Domain")
        self.isopen = True
        self.domain = domain
        self.name = name
        self.index = 0 if index is None else int(index)
        self.start_called = False

        # Create the publisher and subscriber. Both depend on the DDS
        # partition, and so are created here instead of in Domain,
        # where most similar objects are created.
        self.partition_name = os.environ.get("LSST_DDS_DOMAIN")
        if self.partition_name is None:
>           raise RuntimeError("Environment variable $LSST_DDS_DOMAIN not defined")
E           RuntimeError: Environment variable $LSST_DDS_DOMAIN not defined

/opt/lsst/software/stack/stack/tssw/ts_salobj/python/lsst/ts/salobj/sal_info.py:227: RuntimeError
```

Maybe a bug in a test then?

File: conda/meta.yaml (hunk 1 above)
Comment (r-owen) [reply conda/meta.yaml:39]: Thanks for catching that.

**Comment by r-owen** [comment]
Good catch! It's a bug in `ts_salobj`: `BaseCscTestCase` only sets a random `$LSST_DDS_DOMAIN` in the `make_csc` method, which is not called by the `check_bin_script` method. I filed https://jira.lsstcorp.org/browse/DM-25788.

I think most of my CSCs have this problem. I talked to @tribeiro and he is willing to leave the bug for now, with the understanding that it will be fixed soon in ts_salobj.

**Review by tribeiro — APPROVED** [review]

**Pull request merged**